l10n_ch_import_camt054/res/camt.054.demo-1000.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Document xmlns="urn:iso:std:iso:20022:tech:xsd:camt.054.001.04" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:iso:std:iso:20022:tech:xsd:camt.054.001.04 camt.054.001.04.xsd">
+  <BkToCstmrDbtCdtNtfctn>
+    <GrpHdr>
+      <MsgId>20171030375204003295156</MsgId>
+      <CreDtTm>2017-10-31T01:39:34</CreDtTm>
+      <MsgPgntn>
+        <PgNb>1</PgNb>
+        <LastPgInd>true</LastPgInd>
+      </MsgPgntn>
+      <AddtlInf>Productive</AddtlInf>
+    </GrpHdr>
+    <Ntfctn>
+      <Id>20171030375204003295155</Id>
+      <CreDtTm>2017-10-31T01:39:34</CreDtTm>
+      <FrToDt>
+        <FrDtTm>2017-10-28T00:00:00</FrDtTm>
+        <ToDtTm>2017-10-30T23:59:59</ToDtTm>
+      </FrToDt>
+      <Acct>
+        <Id>
+          <IBAN>CH0709000000173125620</IBAN>
+        </Id>
+        <Ownr>
+          <Nm>COMPASSION SUISSE Yverdon-les-Bains</Nm>
+        </Ownr>
+      </Acct>
+      <Ntry>     
+        <NtryRef>010444437</NtryRef>
+        <Amt Ccy="CHF">1000</Amt>
+        <CdtDbtInd>CRDT</CdtDbtInd>
+        <RvslInd>false</RvslInd>
+        <Sts>BOOK</Sts>
+        <BookgDt>
+          <Dt>2017-10-30</Dt>
+        </BookgDt>
+        <ValDt>
+          <Dt>2017-10-30</Dt>
+        </ValDt>
+        <AcctSvcrRef>99999999999999999999999999999999</AcctSvcrRef>
+        <BkTxCd>
+          <Domn>
+            <Cd>PMNT</Cd>
+            <Fmly>
+              <Cd>IDDT</Cd>
+              <SubFmlyCd>PMDD</SubFmlyCd>
+            </Fmly>
+          </Domn>
+        </BkTxCd>
+        <NtryDtls>
+          <Btch>
+            <NbOfTxs>1</NbOfTxs>
+          </Btch>
+          <TxDtls>
+            <Refs>
+              <AcctSvcrRef>99999999999999999999999999999999</AcctSvcrRef>
+              <Prtry>
+              <Tp>04</Tp>
+                <Ref>20171027149402000100037</Ref>
+              </Prtry>
+            </Refs>
+            <Amt Ccy="CHF">400</Amt>
+            <CdtDbtInd>CRDT</CdtDbtInd>
+            <BkTxCd>
+              <Domn>
+                <Cd>PMNT</Cd>
+                <Fmly>
+                  <Cd>IDDT</Cd>
+                  <SubFmlyCd>PMDD</SubFmlyCd>
+                </Fmly>
+              </Domn>
+            </BkTxCd>
+            <RltdPties>
+              <Dbtr>
+                <Nm>Wenk Timothy</Nm>
```

Comment:
You should not include real names ! Please change to fake data, same for bank accounts and addresses!